models/model.js

```diff
@@ -8,3 +9,10 @@ exports.getTopicsModel = () => {
     return rows;
   });
 };
+
+exports.getEndpointsModel = () => {
+  console.log("here");
+  return fs.readFile("./endpoints.json", "utf8").then((data) => {
+    return data;
+  });
```

Comment:
This is reading a json file to a string, serving back a string to the controller, and the controller is serving back a string to the client...your test is parsing it.

The goal of the endpoints.json is to server up a json object to anyone consuming your api (eg you) to know what is available to use in an easy to consume format. This will currently display an inline string in browser and be incomprehensible.